pkg/reconciler/pipelinerun/affinity_assistant_test.go

```diff
@@ -189,23 +191,6 @@ func TestCreateAndDeleteOfAffinityAssistant(t *testing.T) {
 					t.Errorf("unexpected error when retrieving StatefulSet which expects nil: %v", err)
 				}
 			}
-
```

Comment:
Hmmm, after giving it a bit more I now think it makes sense to keep the code here 🙃, as it covers the end-to-end from creation to deletion scenario. We don't necessarily need an E2E test for it if it can be covered by ut. Changed it back!